File: solutions/3188-find-champion-i/find-champion-i.js
/**
 * @param {number[][]} grid
 * @return {number}
 */
var findChampion = function(grid) {
    let winner = 0;
    const n = grid.length;
    for (let opponent = 0; opponent < n; opponent++) {
        if (opponent === winner) continue;
        if (grid[winner][opponent] === 0) {
            winner = opponent;
        }
    }
    return winner;
};

File: solutions/3188-find-champion-i/find-champion-i.py
class Solution:
    def findChampion(self, grid: List[List[int]]) -> int:
        winner = 0
        n = len(grid)
        for opponent in range(n):
            if opponent == winner:
                continue
            if grid[winner][opponent] == 0:
                winner = opponent
        return winner